mani_skill/envs/tasks/control/hopper.py

```diff
@@ -0,0 +1,266 @@
+        # TODO (xhin): further tune controller params
+        max_delta = 2.25
+        stiffness = 100
+        friction = 0.1
+        force_limit = 200
+        damping = 7.5
+        pd_joint_delta_pos_body = PDJointPosControllerConfig(
+            ["hip", "knee", "waist"],
+            lower=-max_delta,
+            upper=max_delta,
+            damping=damping,
+            stiffness=stiffness,
+            force_limit=force_limit,
+            friction=friction,
+            use_delta=True,
+        )
+        pd_joint_delta_pos_ankle = PDJointPosControllerConfig(
+            ["ankle"],
+            lower=-max_delta / 3,
+            upper=max_delta / 3,
+            damping=damping,
+            stiffness=stiffness,
+            force_limit=force_limit,
+            friction=friction,
+            use_delta=True,
+        )
+        rest = PassiveControllerConfig(
+            [j.name for j in self.robot.active_joints if "root" in j.name],
+            damping=0,
+            friction=0,
+        )
+        return deepcopy_dict(
+            dict(
+                pd_joint_delta_pos=dict(
+                    body=pd_joint_delta_pos_body,
+                    ankle=pd_joint_delta_pos_ankle,
+                    rest=rest,
+                    balance_passive_force=False,
+                ),
+            )
+        )
+
+    def _load_articulation(self):
+        """
+        Load the robot articulation
+        """
+        loader = self.scene.create_mjcf_loader()
+        asset_path = str(self.mjcf_path)
+
+        loader.name = self.uid
+
+        # only need the robot
+        self.robot = loader.parse(asset_path)[0][0].build()
+        assert self.robot is not None, f"Fail to load URDF/MJCF from {asset_path}"
+        self.robot_link_ids = [link.name for link in self.robot.get_links()]
+
+    # planar agent has root joints in range [-inf, inf], not ideal in obs space
+    def get_proprioception(self):
+        return dict(
+            # don't include xslider qpos, for x trans invariance
+            qpos=self.robot.get_qpos()[:, 1:],
+            qvel=self.robot.get_qvel(),
+        )
+
+
+class HopperEnv(BaseEnv):
+    agent: Union[HopperRobot]
+
+    def __init__(self, *args, robot_uids=HopperRobot, **kwargs):
+        super().__init__(*args, robot_uids=robot_uids, **kwargs)
+
+    @property
+    def _default_sim_config(self):
+        return SimConfig(
+            gpu_memory_cfg=GPUMemoryConfig(
+                found_lost_pairs_capacity=2**25, max_rigid_patch_count=2**18
+            ),
+            scene_cfg=SceneConfig(
+                solver_position_iterations=15, solver_velocity_iterations=15
+            ),
+            # sim_freq=50,
+            control_freq=25,
+        )
+
+    @property
+    def _default_sensor_configs(self):
+        return self.agent._sensor_configs
+
+    @property
+    def _default_human_render_camera_configs(self):
+        return self.agent._sensor_configs
```

Comment:
It would be better to just copy the camera config for the agent sensor and put it here. You also should change the camera name and make the default height/width of the camera to be larger (512x512)

Reply:
fixed :+1: :+1: :+1: